json/src/main/java/com/basistech/rosette/apimodel/jackson/EntityMentionMixin.java

```diff
@@ -19,15 +19,17 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class EntityMixin extends BaseMixin {
+public class EntityMentionMixin extends BaseMixin {
     @JsonCreator
-    public EntityMixin(
+    public EntityMentionMixin(
             @JsonProperty("indocChainId") Integer indocChainId,
             @JsonProperty("type") String type,
             @JsonProperty("mention") String mention,
             @JsonProperty("normalized") String normalized,
-            @JsonProperty("count") int count,
-            @JsonProperty("entityId") String entityId
+            @JsonProperty("count") Integer count,
```

Comment:
That is why I marked it deprecated.
On Jun 1, 2016 9:30 PM, "Li Xu" <notifications@github.com> wrote:

> In
> json/src/main/java/com/basistech/rosette/apimodel/jackson/EntityMentionMixin.java
> <https://github.com/rosette-api/java/pull/46#discussion_r65471632>:
>
> >              @JsonProperty("indocChainId") Integer indocChainId,
> >              @JsonProperty("type") String type,
> >              @JsonProperty("mention") String mention,
> >              @JsonProperty("normalized") String normalized,
> > -            @JsonProperty("count") int count,
> > -            @JsonProperty("entityId") String entityId
> > +            @JsonProperty("count") Integer count,
>
> @benson-basis <https://github.com/benson-basis> I just realize that it we
> are no longer deduping the mentions, then this count makes no sense.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rosette-api/java/pull/46/files/9c0d04336c2919029fe2cb370b0c5c215e4f088c#r65471632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADM9z0R8dpNY5eDiTPHgAICrHESff_Ycks5qHjIbgaJpZM4Iq-22>
> .
>